File: golang/apps/app1/cls/person.go
package cls

import (
	"fmt"
	"strconv"
)

type Person struct {
	name    string
	age     int
	country string
}

func NewInstance(name string, age int, country string) *Person {
	p := new(Person)
	p.name = name
	p.age = age
	p.country = country

	return p
}

func (p Person) SayHello() {
	fmt.Println("Who am I?")
	fmt.Println("My name is" + p.name + ".")
	fmt.Println("I'm " + strconv.Itoa(p.age) + " years old.")
	fmt.Println("I'm live in " + p.country + ".")
}

File: csharp/apps/app1/cls/Person.cs
public class Person {
    private string name;
    private int age;
    private string country;

    public Person(string name, int age, string country) {
        this.name = name;
        this.age = age;
        this.country = country;
    }

    public void sayHello() {
        Console.WriteLine("Hello, my name is " + this.name + ".");
        Console.WriteLine("I'm " + this.age + " years old now.");
        Console.WriteLine("I'm live in " + this.country + ".");
    }
}